distributed/scheduler.py

```diff
@@ -6201,31 +6201,27 @@ def adaptive_target(self, comm=None, target_duration=None):
             self.total_occupancy / target_duration
         )  # TODO: threads per worker
 
-        # Avoid a few long tasks from asking for many cores
-        ws: WorkerState
-        tasks_processing = 0
-        for ws in self.workers.values():
-            tasks_processing += len(ws._processing)
-
-            if tasks_processing > cpu:
-                break
-        else:
-            cpu = min(tasks_processing, cpu)
```

Comment:
this code prevented more worker to spawn for me. It is not needed anymore as the code now limits the number of targeted workers by the number of runnable tasks.